crossfit/backend/torch/hf/model.py

```diff
@@ -81,11 +96,13 @@ def fit_memory_estimate_curve(self, model=None):
         y = []
 
         max_seq = self.max_seq_length()
-        for batch_size in tqdm(range(2048, 0, -256)):
+        for batch_size in tqdm(
+            range(self.end_batch_size, self.start_batch_size, -self.batch_size_increment)
```

Comment:
Might be best to do
```suggestion
            range(self.end_batch_size, self.start_batch_size - 1, -self.batch_size_increment)
```

Otherwise with `self.start_batch_size = 1` you won't actually hit the batch size of 1

Reply:
Resolved by: https://github.com/rapidsai/crossfit/pull/66/commits/bbdc4f2dec05cd31168bb8e0b2e9ad62ec586e54